<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectra Subtraction</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="rowSkipSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="specialValueText">
     <string>6</string>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="prefix">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rowLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rows to skip:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="buddy">
     <cstring>rowSkipSpinBox</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="mainSpectraLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Main Spectra</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="buddy">
     <cstring>mainSpectraBox</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="loadfileButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="mainSpectraBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="graphLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Graphs to Plot</string>
    </property>
    <property name="buddy">
     <cstring>graphsWidget</cstring>
    </property>
   </widget>
   <widget class="QListWidget" name="graphsWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>261</width>
      <height>301</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="subtractBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="subtractLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtract from Main</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="buddy">
     <cstring>subtractBox</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="plotSubtractionButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Subtraction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFolderButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Save Location</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveLocationLineEdit">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>550</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="saveLocationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Save Location</string>
    </property>
    <property name="buddy">
     <cstring>saveLocationLineEdit</cstring>
    </property>
   </widget>
   <widget class="QCheckBox" name="toggleNormalization">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>490</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalized</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="saveGraphBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Graphs</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="peaksAnnotate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="specialValueText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Peaks to Annotate</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="buddy">
     <cstring>peaksAnnotate</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="plotGraphs">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plotDualButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Dual</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Dual Spectra&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>A-B on top B-A on bottom</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Spectra A</string>
    </property>
    <property name="buddy">
     <cstring>spectraABox</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="spectraABox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="spectraBBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Spectra B</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSpectra_Subtraction">
    <property name="title">
     <string>Spectra Subtraction</string>
    </property>
   </widget>
   <addaction name="menuSpectra_Subtraction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_to">
   <property name="text">
    <string>Save to ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
